metadata.yaml

```diff
@@ -0,0 +1,25 @@
+# Copyright 2021 Daniela Plascencia
+# See LICENSE file for licensing details.
+
+# For a complete list of supported options, see:
```

Comment:
We can probably also just remove this comment

Reply:
Oops, didn't notice it. Will remove it.